src/main/java/org/datadog/jmxfetch/reporter/StatsdReporter.java

```diff
@@ -28,7 +28,7 @@ protected void sendMetricPoint(String metricName, double value, String[] tags) {
             this.statsDClient.stop();
             init();
         }
-        statsDClient.gauge(metricName, value, tags);
+        statsDClient.gauge(StatsdReporter.formatDataName(metricName), value, tags);
```

Comment:
Or actually make it so the service check name is also read from the configuration, just like the metrics are... what do you think ?